run_unit_tests.sh

```diff
@@ -44,6 +44,7 @@ nose_args=$@
 which airflow > /dev/null || python setup.py develop
 
 echo "Initializing the DB"
+airflow initdb
```

Comment:
Isn;t this redundant with below?

Reply:
I was running into some strangeness where it would attempt to reset a db that had not yet been initialized.